include/networkit/graph/Graph.hpp

```diff
@@ -989,7 +1004,7 @@ class Graph final {
      * @param v Node.
      * @return The volume of the @a v.
      */
-    edgeweight volume(node v) const;
+    edgeweight TLX_DEPRECATED(volume(node v) const);
```

Comment:
I must admit that I didn't know the definition of an node's volume (but in my defence and more to the point, it doesn't appear in [Wikipedia's Glossary of graph theory terms](https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms)).  If we move the function anyhow, can we please rename it to something more verbose, e.g. `totalEdgeWeights`, `edgeWeightSum`, ... ?

Reply:
I just realized that there's already a `Graph::computeWeightedDegree`. Can't we just add the flag `countSelfloopTwice`?

Reply:
Yes, now `volume` returns `weightedDegree` with self-loops counted twice.